dds/DCPS/XTypes/TypeLookupService.h

```diff
@@ -21,12 +21,24 @@
 #pragma once
 #endif /* ACE_LACKS_PRAGMA_ONCE */
 
+#ifdef OPENDDS_SAFETY_PROFILE
+namespace DDS {
+  enum ExtensibilityKind {
```

Comment:
Should this come from the IDL?  Maybe move it from DdsDynamicData to a common IDL file that's already included?

Reply:
This may not be needed anymore.  I'll see if that is the case.